ci/build.ps1

```diff
@@ -98,27 +99,17 @@ npm.cmd install -g neovim
 Get-Command -CommandType Application neovim-node-host.cmd
 npm.cmd link neovim
 
-#npm.cmd install -g tree-sitter-cli
-#npm.cmd link tree-sitter-cli
+npm.cmd install -g 'tree-sitter-cli@0.15.9'
 
-mkdir c:\treesitter
-$env:TREE_SITTER_DIR = "c:\treesitter"
-#$env:PATH = "c:\treesitter;$env:PATH"
+$env:TREE_SITTER_DIR = $env:USERPROFILE + "\tree-sitter-build"
+mkdir "$env:TREE_SITTER_DIR\bin"
 $client = new-object System.Net.WebClient
-cd c:\treesitter
 
-if ($bits -eq 32) {
-  $client.DownloadFile("https://github.com/tree-sitter/tree-sitter/releases/download/0.15.9/tree-sitter-windows-x86.gz", "c:\treesitter\tree-sitter-cli.gz")
-}
-elseif ($bits -eq 64) {
-  $client.DownloadFile("https://github.com/tree-sitter/tree-sitter/releases/download/0.15.9/tree-sitter-windows-x64.gz", "c:\treesitter\tree-sitter-cli.gz")
-}
-python -c "import gzip, shutil; f1,f2 = gzip.open('tree-sitter-cli.gz', 'rb'), open('tree-sitter.exe', 'wb'); shutil.copyfileobj(f1, f2); f2.close()"
-
-$client.DownloadFile("https://codeload.github.com/tree-sitter/tree-sitter-c/zip/v0.15.2","c:\treesitter\tree_sitter_c.zip")
-Expand-Archive c:\treesitter\tree_sitter_c.zip -DestinationPath c:\treesitter\
-cd c:\treesitter\tree-sitter-c-0.15.2
-c:\treesitter\tree-sitter.exe test
+$client.DownloadFile("https://codeload.github.com/tree-sitter/tree-sitter-c/zip/v0.15.2",".\tree_sitter_c.zip")
+Expand-Archive .\tree_sitter_c.zip -DestinationPath .
+cd tree-sitter-c-0.15.2
+tree-sitter.cmd test
+Test-Path -PathType Leaf "$env:TREE_SITTER_DIR\bin\c.dll"
```

Comment:
This passes but I don't get why the following happens:

```
[  ERROR   ] test/functional\lua\treesitter_spec.lua @ 39: treesitter API with C parser before_each
test\functional\helpers.lua:106: Error executing lua: Failed to load parser: uv_dlopen: C:\Users\appveyor\tree-sitter-build/bin/c.dll is not a valid Win32 application
```

Reply:
- is `uv_dlopen` is being strict about slashes (unlikely)? No, a similar mixed-slashes path is constructed by `treesitter_spec.lua`, and it passes on master.
- does `c.dll` needs to be sibling to the `.exe` or some other files? I don't see any changes here which would result in different contents of `bin/`.

Perhaps it's worth trying the old `c:\treesitter` path.

Reply:
https://github.com/neovim/neovim/pull/10124#issuecomment-504610136

It has to do with VM arch on Appveyor.